rdmo/core/checks.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.core.checks import Error, register
+
+
+@register()
```

Comment:
This is nice, we should use this more often.

Reply:
yes, should I already refactor into a package `rdmo/core/checks/accounts.py` ?

In another PR, I would also like to add a check for when `DEBUG` is true and the password hasher is not set correctly  :)

Reply:
I would keep it in core for now, technically all `settings` are kind of `core`.

Reply:
And there is a case for `DEBUG` on without the test fixture and real users so ... no.

Reply:
no, leave it like it is, we can refactor if the file gets to big. (In daiquiri I moved settings files to all apps btw.).

Yes a warning would be cool.